### PR TITLE
Fix lack of target power control in BMDA

### PR DESCRIPTION
**Pull request by sidprice**
BMDA was not controlling the target power when the "--power" option was used.

* [x] I've read the [Code of Conduct](https://github.com/blackmagic-debug/blackmagic/blob/main/CODE_OF_CONDUCT.md)
* [x] I've read the [guidelines for contributing](https://github.com/blackmagic-debug/blackmagic/blob/main/CONTRIBUTING.md) to this repository
* [ ] It builds for hardware native (`make PROBE_HOST=native`)
* [x] It builds as BMDA (`make PROBE_HOST=hosted`)
* [x] I've tested it to the best of my ability
* [x] My commit messages provide a useful short description of what the commits do

### CODE REVIEW

File: src/platforms/hosted/bmp_remote.c
Comment (dragonmux) [inline src/platforms/hosted/bmp_remote.c:49]: Please remote the extra space between `)` and `;` (this will be the PR lint check issue)

**Review by dragonmux — CHANGES_REQUESTED** [review]
There's just one formatting issue which once fixed, we'll happily merge this PR

**Comment by sidprice** [comment]
I didn't save the source before I built it, so in VSCode the linter didn't

**Review by dragonmux — APPROVED** [review]
LGTM, we'll set it to merge once the checks complete

**Pull request merged**
